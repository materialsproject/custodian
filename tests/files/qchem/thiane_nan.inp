$molecule
 -1 2
 C          0.62598       -0.33426       -0.64130
 C          0.00432       -1.32250        0.37722
 H          0.89858       -0.93181       -1.52615
 C         -1.46053       -1.63063        0.04823
 H          0.58378       -2.25941        0.36787
 H          0.07046       -0.92272        1.39672
 H         -1.78991       -2.53282        0.58916
 H         -1.56390       -1.86459       -1.01978
 C         -0.38146        0.71452       -1.17198
 H          0.14167        1.64834       -1.41093
 H         -0.82807        0.35178       -2.10489
 C         -2.39622       -0.48145        0.43453
 H         -3.38681       -0.62230       -0.01916
 H         -2.52431       -0.45859        1.52430
 S         -1.74966        1.18990       -0.03475
 N          1.89226        0.30196       -0.22497
 C          2.99895       -0.63505       -0.11599
 H          3.05228       -1.26242       -1.01625
 H          3.94340       -0.07700       -0.03384
 H          2.93295       -1.30899        0.76201
 C          1.82245        1.18277        0.93354
 H          2.75550        1.76325        0.99577
 H          0.98642        1.88146        0.83291
 H          1.70068        0.64878        1.89708
$end

$rem
   BASIS  =  6-31+G*
   EXCHANGE  =  B3LYP
   JOB_TYPE  =  Frequency
   MEM_TOTAL = 2000
$end
