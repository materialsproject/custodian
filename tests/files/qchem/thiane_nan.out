Process 0 of 1 is on c0937 - thread support 0
                  Welcome to Q-Chem
     A Quantum Leap Into The Future Of Chemistry

 Y. Shao, L. Fusti-Molnar, Y. Jung, J. Kussmann, C. Ochsenfeld,
 S. T. Brown, A. T. B. Gilbert, L. V. Slipchenko,S. V. Levchenko,
 D. P. O'Neill, R. A. DiStasio Jr., R. C. Lochan, T. Wang,
 G. J. O. Beran, N. A. Besley, J. M. Herbert, C. Y. Lin,
 T. Van Voorhis, S. H. Chien, A. Sodt, R. P. Steele, 
 V. A. Rassolov, P. E. Maslen, P. P. Korambath, R. D. Adamson,
 B. Austin, J. Baker, E. F. C. Byrd, H. Dachsel, R. J. Doerksen,
 A. Dreuw, B. D. Dunietz, A. D. Dutoi, T. R. Furlani,
 S. R. Gwaltney, A. Heyden, S. Hirata, C.-P. Hsu, G. Kedziora,
 R. Z. Khaliullin, P. Klunzinger, A. M. Lee, M. S. Lee, W. Liang,
 I. Lotan, N. Nair, B. Peters, E. I. Proynov, P. A. Pieniazek, 
 Y. M. Rhee, J. Ritchie, E. Rosta, C. D. Sherrill, 
 A. C. Simmonett, J. E. Subotnik, H. L. Woodcock III,
 W. Zhang, A. T. Bell, A. K. Chakraborty, D. M. Chipman,
 F. J. Keil, A. Warshel, W. J. Hehre, H. F. Schaefer III,
 J. Kong, A. I. Krylov, P. M. W. Gill, M. Head-Gordon,
 Q-Chem, Version 3.1, Q-Chem, Inc., Pittsburgh, PA (2007).

 Additional authors for Version 3.1: 
 Z. Gan, Y. Zhao, N. E. Schultz, D. Truhlar, E. Epifanovsky and M. Oana.

 Additional authors for Version 3.2: 
 R. Baer, B. R. Brooks, D. Casanova, J.-D. Chai, C.-L. Cheng, 
 C. Cramer, D. Crittenden, A. Ghysels, G. Hawkins, E. G. Hohenstein, 
 C. Kelley, W. Kurlancheek, D. Liotard, E. Livshits, P. Manohar, 
 A. Marenich, D. Neuhauser, R. Olson, M. A. Rohrdanz, K. S. Thanthiriwatte, 
 A. J. W. Thom, V. Vanovschi, C. F. Williams, Q. Wu and Z.-Q. You. 


 Intel X86 EM64T Linux Version

 Q-chem begins on Wed Oct 30 22:18:49 2013  

 Parallel job on  1  processors
Processing $rem in /usr/common/usg/qchem/3.2.0.2/config/preferences.

theFileMan(): MAXOPENFILES=974 MAX_SUB_FILE_NUM=16
Maximum size of a physical file is 2.0 GB, maximum size of a tmp-file is 32.0 GB


--------------------------------------------------------------
User input:
--------------------------------------------------------------
$molecule
 -1 2
 C          0.62598       -0.33426       -0.64130
 C          0.00432       -1.32250        0.37722
 H          0.89858       -0.93181       -1.52615
 C         -1.46053       -1.63063        0.04823
 H          0.58378       -2.25941        0.36787
 H          0.07046       -0.92272        1.39672
 H         -1.78991       -2.53282        0.58916
 H         -1.56390       -1.86459       -1.01978
 C         -0.38146        0.71452       -1.17198
 H          0.14167        1.64834       -1.41093
 H         -0.82807        0.35178       -2.10489
 C         -2.39622       -0.48145        0.43453
 H         -3.38681       -0.62230       -0.01916
 H         -2.52431       -0.45859        1.52430
 S         -1.74966        1.18990       -0.03475
 N          1.89226        0.30196       -0.22497
 C          2.99895       -0.63505       -0.11599
 H          3.05228       -1.26242       -1.01625
 H          3.94340       -0.07700       -0.03384
 H          2.93295       -1.30899        0.76201
 C          1.82245        1.18277        0.93354
 H          2.75550        1.76325        0.99577
 H          0.98642        1.88146        0.83291
 H          1.70068        0.64878        1.89708
$end

$rem
   BASIS  =  6-31+G*
   EXCHANGE  =  B3LYP
   JOB_TYPE  =  Frequency
   MEM_TOTAL = 2000
$end

--------------------------------------------------------------
 Writing REM_CC_EA             0
 ----------------------------------------------------
       Standard Nuclear Orientation (Angstroms)
    I     Atom         X            Y            Z
 ----------------------------------------------------
    1      C       0.627120    -0.312543    -0.637253
    2      C       0.025150    -1.335869     0.358186
    3      H       0.893448    -0.880987    -1.542944
    4      C      -1.441237    -1.645326     0.037389
    5      H       0.611393    -2.267531     0.313631
    6      H       0.100620    -0.966524     1.388462
    7      H      -1.757359    -2.565961     0.554633
    8      H      -1.555749    -1.847757    -1.035913
    9      C      -0.394409     0.743896    -1.123991
   10      H       0.118854     1.688543    -1.340752
   11      H      -0.849541     0.406000    -2.062097
   12      C      -2.380700    -0.515641     0.469335
   13      H      -3.375621    -0.650446     0.023351
   14      H      -2.495795    -0.526665     1.560738
   15      S      -1.752275     1.174087     0.043222
   16      N       1.893575     0.320665    -0.216883
   17      C       3.008424    -0.610564    -0.149416
   18      H       3.055540    -1.210055    -1.068816
   19      H       3.949629    -0.047912    -0.061605
   20      H       2.958030    -1.311178     0.708509
   21      C       1.831216     1.165554     0.968494
   22      H       2.760615     1.751137     1.037209
   23      H       0.988869     1.860433     0.898956
   24      H       1.725051     0.601807     1.916809
 ----------------------------------------------------
 Molecular Point Group                 C1    NOp =  1
 Largest Abelian Subgroup              C1    NOp =  1
 Nuclear Repulsion Energy =   579.4333861202 hartrees
 There are       41 alpha and       40 beta electrons
 Requested basis set is 6-31+G(d)
 There are 76 shells and 205 basis functions

Total QAlloc Memory Limit   2000 MB
Mega-Array Size        77 MB
MEM_STATIC part        62 MB
 A cutoff of  1.0D-11 yielded   2706 shell pairs
 There are  20414 function pairs
Evaluating contribution to one-electron hamiltonian from nuclear
 Smallest overlap matrix eigenvalue = 1.51E-05
 Multipole matrices computed through 2nd order
 Guess from superposition of atomic densities
 Warning:  Energy on first SCF cycle will be non-variational
 An unrestricted hybrid HF-DFT SCF calculation will be
 performed using Pulay DIIS extrapolation
 Exchange:     0.2000 Hartree-Fock + 0.0800 Slater + 0.7200 Becke88
 Correlation:  0.8100 LYP + 0.1900 VWN1RPA
 Using SG-1 standard quadrature grid
 SCF converges when DIIS error is below 1.0E-08
 Warning!  Inaccurate integrated density:
   Number of electrons =    81
   Numerical integral  =    79.998799555684
   Relative error      =    -1.2360499313 %
 ---------------------------------------
  Cycle       Energy         DIIS Error
 ---------------------------------------
    1    -730.1931991222      4.75E-02
    2    -728.3799593005      6.04E-03
    3    -727.9594428992      1.01E-02
    4    -728.6542783236      1.40E-03
    5    -728.6657528732      9.33E-04
    6    -728.6721146885      2.88E-04
    7    -728.6729949738      1.48E-04
    8    -728.6733697481      5.27E-05
    9    -728.6735520662      2.57E-05
   10    -728.6736307341      1.83E-05
   11    -728.6736924949      1.11E-05
   12    -728.6737076543      4.38E-06
   13    -728.6737082578      3.04E-06
   14    -728.6737084742      1.40E-06
   15    -728.6737085669      7.12E-07
   16    -728.6737085973      3.16E-07
   17    -728.6737086003      1.36E-07
   18    -728.6737086013      4.40E-08
   19    -728.6737086016      2.53E-08
   20    -728.6737086015      1.18E-08
   21    -728.6737086010      3.71E-09 Convergence criterion met
 ---------------------------------------
 <S^2> = 0.7501
 SCF time:  CPU 375.25 s  wall 375.25 s
 Analysis of SCF Wavefunction

 --------------------------------------------------------------
                    Orbital Energies (a.u.)
 --------------------------------------------------------------

 Alpha MOs
 -- Occupied --
-88.748 -14.209 -10.095 -10.092 -10.089 -10.084 -10.083 -10.064
-10.054  -7.808  -5.773  -5.770  -5.763  -0.801  -0.703  -0.629
 -0.621  -0.585  -0.529  -0.504  -0.463  -0.413  -0.372  -0.358
 -0.340  -0.327  -0.311  -0.294  -0.277  -0.271  -0.268  -0.254
 -0.244  -0.237  -0.227  -0.217  -0.199  -0.171  -0.101  -0.088
  0.076
 -- Virtual --
  0.101   0.107   0.115   0.120   0.129   0.137   0.140   0.147
  0.156   0.165   0.175   0.179   0.182   0.184   0.195   0.200
  0.205   0.206   0.214   0.217   0.222   0.230   0.234   0.248
  0.256   0.260   0.267   0.269   0.276   0.281   0.286   0.291
  0.301   0.305   0.329   0.337   0.343   0.344   0.359   0.362
  0.369   0.373   0.375   0.384   0.390   0.397   0.407   0.411
  0.428   0.443   0.455   0.468   0.470   0.480   0.497   0.503
  0.518   0.546   0.566   0.626   0.628   0.664   0.685   0.719
  0.740   0.753   0.784   0.788   0.804   0.809   0.822   0.827
  0.849   0.860   0.872   0.892   0.902   0.926   0.959   0.979
  0.993   0.997   1.020   1.047   1.066   1.081   1.091   1.101
  1.107   1.111   1.125   1.135   1.140   1.155   1.165   1.180
  1.210   1.217   1.239   1.248   1.255   1.265   1.281   1.290
  1.297   1.315   1.356   1.376   1.434   1.530   1.559   1.575
  1.600   1.635   1.700   1.707   1.777   1.802   1.864   1.881
  1.966   1.995   2.016   2.025   2.053   2.065   2.088   2.135
  2.167   2.175   2.186   2.202   2.242   2.273   2.300   2.311
  2.334   2.363   2.372   2.393   2.417   2.439   2.470   2.515
  2.551   2.561   2.649   2.673   2.724   2.801   2.839   2.855
  2.969   2.976   3.005   4.066   4.280   4.389   4.419   4.482
  4.495   4.533   4.674   4.846

 Beta MOs
 -- Occupied --
-88.748 -14.208 -10.095 -10.092 -10.088 -10.083 -10.083 -10.063
-10.054  -7.808  -5.773  -5.770  -5.763  -0.800  -0.702  -0.628
 -0.620  -0.584  -0.528  -0.503  -0.462  -0.412  -0.371  -0.357
 -0.340  -0.327  -0.311  -0.293  -0.277  -0.270  -0.268  -0.253
 -0.244  -0.237  -0.226  -0.216  -0.199  -0.170  -0.100  -0.087
 -- Virtual --
  0.102   0.109   0.111   0.119   0.126   0.131   0.143   0.145
  0.148   0.158   0.167   0.176   0.181   0.184   0.186   0.197
  0.203   0.207   0.208   0.216   0.219   0.224   0.231   0.237
  0.250   0.257   0.263   0.269   0.271   0.279   0.283   0.288
  0.293   0.302   0.308   0.330   0.339   0.344   0.345   0.360
  0.364   0.371   0.374   0.377   0.385   0.390   0.398   0.408
  0.412   0.429   0.445   0.457   0.469   0.472   0.482   0.498
  0.504   0.519   0.548   0.567   0.626   0.628   0.665   0.686
  0.719   0.741   0.754   0.785   0.789   0.805   0.809   0.822
  0.828   0.849   0.861   0.872   0.893   0.903   0.926   0.959
  0.979   0.994   0.997   1.021   1.047   1.067   1.082   1.092
  1.101   1.107   1.111   1.126   1.136   1.140   1.155   1.166
  1.182   1.211   1.218   1.240   1.249   1.256   1.265   1.282
  1.291   1.298   1.316   1.356   1.376   1.434   1.530   1.560
  1.576   1.600   1.635   1.700   1.708   1.777   1.802   1.865
  1.881   1.967   1.995   2.017   2.025   2.054   2.065   2.088
  2.135   2.167   2.176   2.187   2.203   2.242   2.274   2.300
  2.312   2.334   2.363   2.373   2.394   2.418   2.440   2.470
  2.516   2.552   2.562   2.650   2.674   2.725   2.801   2.840
  2.856   2.969   2.976   3.005   4.066   4.280   4.389   4.419
  4.483   4.495   4.534   4.675   4.847
 --------------------------------------------------------------

          Ground-State Mulliken Net Atomic Charges

     Atom                 Charge (a.u.)    Spin (a.u.)
  --------------------------------------------------------
      1 C                    -1.729651       1.149339
      2 C                     0.936506      -0.938553
      3 H                     0.204343       0.000567
      4 C                     1.615614      -2.097728
      5 H                     0.195028       0.022787
      6 H                     0.200713       0.008262
      7 H                     0.192572       0.021455
      8 H                     0.203533       0.020443
      9 C                    -0.567307      -0.222624
     10 H                     0.234458       0.011041
     11 H                     0.243361       0.008607
     12 C                    -2.141744       1.479562
     13 H                     0.223180       0.001514
     14 H                     0.221409       0.003087
     15 S                    -0.691067       0.913616
     16 N                     0.049607      -0.003211
     17 C                    -0.780040       0.357048
     18 H                     0.202386      -0.000706
     19 H                     0.178501       0.002615
     20 H                     0.186450       0.007091
     21 C                    -0.754628       0.250807
     22 H                     0.181309       0.001383
     23 H                     0.208712       0.003368
     24 H                     0.186754       0.000230
  --------------------------------------------------------
  Sum of atomic charges =    -1.000000
  Sum of spin   charges =     1.000000

 -----------------------------------------------------------------
                    Cartesian Multipole Moments
 -----------------------------------------------------------------
    Charge (ESU x 10^10)
                -4.8032
    Dipole Moment (Debye)
         X       0.0340      Y       3.3697      Z       1.4110
       Tot       3.6533
    Quadrupole Moments (Debye-Ang)
        XX    -104.1618     XY      -0.4757     YY    -101.4530
        XZ      -1.5194     YZ      -1.1090     ZZ     -89.9595
    Octapole Moments (Debye-Ang^2)
       XXX     -24.5973    XXY      19.9207    XYY      14.6210
       YYY      81.3174    XXZ      12.7799    XYZ      -3.0255
       YYZ       4.3387    XZZ      -4.6383    YZZ      19.8673
       ZZZ      14.8568
    Hexadecapole Moments (Debye-Ang^3)
      XXXX   -2052.2669   XXXY     -22.4507   XXYY    -533.4761
      XYYY     -55.2257   YYYY   -1038.3447   XXXZ     -16.0014
      XXYZ      -5.8269   XYYZ     -14.5415   YYYZ     -16.3177
      XXZZ    -406.5611   XYZZ      -7.1940   YYZZ    -261.5210
      XZZZ     -19.7096   YZZZ      -6.8858   ZZZZ    -533.4971
 -----------------------------------------------------------------
 Calculating MO derivatives via CPSCF
 Warning:  Singularity detected in AXeqB0            1            1 
            1                       NaN            3

 Q-Chem fatal error occurred in module 0, line  119, PE  0:

 Unable to find relaxed density

